javascript/README.md

```diff
@@ -286,3 +295,137 @@ const { a, ...noA } = copy; // noA => { b: 2, c: 3 }
 - JSPerf: [Shallow Copy Objects][JSPerf: Shallow Copy Objects]
 
 [⇧ top](#javascript-guide)
+
+---
+
+## Strings
+
+### 5.1 Quotes
+
+Use single quotes `''` for strings.
+
+#### Examples
+
+🚫 Nope. 🚫
+
+```js
+const name = "Cloud Four";
+
+// Template literals should contain interpolation or newlines.
+const name = `Cloud Four`;
+```
+
+🎉 Yep! 🎉
+
+```js
+const name = 'Cloud Four';
+```
+
+#### Resources
+
+- ESLint: [quotes]
+
+### 5.2 Line Length
+
+Strings that cause the line to go over 80 characters should not be written across multiple lines using string concatenation.
+
+> Why? Broken strings are painful to work with and make code less searchable.
+
+#### Examples
+
+🚫 Nope. 🚫
+
+```js
+const errorMessage = 'This is a super long error that was thrown because \
+of Batman. When you stop to think about how Batman had anything to do \
+with this, you would get nowhere \
+fast.';
+
+const errorMessage = 'This is a super long error that was thrown because ' +
+  'of Batman. When you stop to think about how Batman had anything to do ' +
+  'with this, you would get nowhere fast.';
+```
+
+🎉 Yep! 🎉
+
+```js
+const errorMessage = 'This is a super long error that was thrown because of Batman. When you stop to think about how Batman had anything to do with this, you would get nowhere fast.';
```

Comment:
I'd like to have strings ignored for `max-len`. It does make searching harder. Any arguments against?

https://eslint.org/docs/rules/max-len#ignorestrings

Reply:
Instead of having specific stylistic details in this guide, can we just mention somewhere in here something like:

> Code should conform to [Prettier](https://prettier.io/) formatting

Reply:
I am removing stylistic sections from this guide for now.